src/Symfony/Component/Notifier/Bridge/Mailjet/README.md

```diff
@@ -0,0 +1,23 @@
+Mailjet Notifier
+================
+
+Provides [Mailjet](https://mailjet.com) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+MAILJET_DSN=mailjet://AUTH_TOKEN@default?from=0611223344
```

Comment:
Maybe detail that it is the sms api, not the mail api
Also what if someone add the mail api part, will it overlap?

Reply:
Mail api part is already used in Mailjet Symfony Mailer Bridge, and it used MAILER_DSN as suggested env variable